### DISCUSSION TITLE
Any time I try to connect to search console I get the above error.

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#3032] [Self-Hosted Support]
Any time I try to connect to search console I get the above error.

The account I connect to have 5 verified websites (properties). All working fine. But when I try to connect in plausible, i get this:

![image](https://github.com/plausible/analytics/assets/3120151/306f755f-6f4f-4c4f-8af0-3bc374794c59)

_Originally posted by @jesperordrup in https://github.com/plausible/analytics/discussions/3008_

**Comment by jesperordrup** [comment]
Ive enabled sentry and get no errors when trying.

**Comment by jesperordrup** [comment]
There is prob no need to create a new discussion as it already exist here #3008. The discussion I originally created because I felt it wasnt an isure but more me needing to understand something  better. 

But the discussion #3008 was fruitless. I got a few pointers and followed them but no effect. This led me to think there is an issue in the solution and I escalated and created a new issue #3031. The issue is now turned into a new discussion (this one) and that I dont understand.

Please, what can I do?

**Reply by ruslandoga** [reply]
👋 @jesperordrup 

This https://github.com/plausible/analytics/discussions/3008#discussioncomment-6100399 is the best support we can provide for self-hosted setups. Sorry.

**Reply by ruslandoga** [reply]
To expand on it a bit, here's how self-hosted support works right now:

- if I know the answer I try to post it right away, sometimes I even end up being correct
- if I don't know the answer (and I don't know much about Plausible's 3rd party integrations like with GA, `tracker.js`, or anything besides the backend code), I can only provide pointers and hope the user would figure out the answer on their own

But in neither case does the core team get involved since they are busy shipping new features and keeping the Cloud offering running. I understand that it sucks to not get the answer you want but it's just a matter of limited knowledge in my case and limited time in core team's case.

Sorry, again.

**Reply by jesperordrup** [reply]
That is totally understandable. I definatly dont expect support directly. i'd like to think that I in a small way help improving the product that you also sell by reporting issues and Im hoping that we / you / community use the issues reported to try fixing this. Thats why I created the issue. This allows others to say "hey - im experiencing the same" and then it might become important to look into.

Edit: I suggest reopening issue
